sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/CreateCallResult.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.callingserver.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The response payload of the create call operation. */
+@Fluent
+public final class CreateCallResult {
+    /*
```

Comment:
Ideally, these "Result" classes should have all final members/fields and be immutable

Reply:
All result classes are made immutable